js/app/Visualization/UI/BasicSidebar.js

```diff
@@ -307,6 +307,81 @@ define([
         slider.startup();
       }
 
+      /* Transplanted from
+       * js/app/Visualization/UI/SidePanels/DataViewUI.js This is a
+       * hack for now... */
+
+      if (!animation.data_view || !animation.data_view.selections.selections.active_category) return;
+      var selection = animation.data_view.selections.selections.active_category;
+      var name = "active_category";
+
+      if (selection.hidden) return;
+      if (selection.sortcols.length != 1) return;
+      var sourcename = selection.sortcols[0]
+      var source = animation.data_view.source.header.colsByName[sourcename];
+      if (!source || !source.choices) return;
+
+      var selectionwidget = new ContentPane({
+        content: "<div>" + name + " from " + sourcename + "</div>",
+        style: "padding-top: 0; padding-bottom: 0;"
+      });
+
+      var selectionselect = $("<div class='imageselector' style='position: fixed; z-index: 100000; background: white; border: 1px solid red;'><div class='choices' style='overflow: auto; height:200px;'></div><div class='label'></div></div>");
+      var choices = Object.keys(
+        CountryCodes.codeToName
+      ).filter(function (key) {
+        return source.choices[key] != undefined;
+      });
+      choices.sort();
+      choices.map(function (key) {
+        var value = source.choices[key];
+        var option = $('<img src="' + app.dirs.img + '/flags/png/' + key.toLowerCase() + '.png" alt="' + CountryCodes.codeToName[key] + '" class="choice">');
+        // option.text(key);
+        option.data("value", value);
+        if (selection.data[sourcename].indexOf(value) != -1) {
+          option.addClass("selected");
+        }
+        option.hover(function () {
+          selectionselect.find('.label').html(CountryCodes.codeToName[key]);
+        }, function () {});
+        option.click(function () {
+          option.toggleClass("selected");
+          updateSelection();
+        });
+        selectionselect.find('.choices').append(option);
+      });
+
+      var toggleselectionselect = $("<div>Countries</div>");
+      toggleselectionselect.click(function () {
+        selectionselect.toggle();
+      });
+      selectionselect.hide();
+      $(self.node.find(".layer-list")).append(toggleselectionselect);
+      $(self.node.find(".layer-list")).append(selectionselect);
+
+      var updateSelection = function () {
+        selection.clearRanges();
+
+        var values = Array.prototype.slice.call(
+          selectionselect.find('.choices .selected')
+        ).map(function (option) {
+          return $(option).data("value");
+        });
+        if (values.length) {
+          values.map(function (value) {
+            var data = {};
+            data[sourcename] = value;
+            selection.addDataRange(data, data);
+          });
+        } else {
+          var startData = {};
+          var endData = {};
+          startData[sourcename] = -1.0/0.0;
```

Comment:
Ha! Feel the Division By Zero exception in your head!
I actually didn't know about these, cool!